core/rio/hdt/src/main/java/org/eclipse/rdf4j/rio/hdt/HDTArrayLog64.java

```diff
@@ -67,7 +82,7 @@ protected void parse(InputStream is) throws IOException {
 		try (UncloseableInputStream uis = new UncloseableInputStream(is);
 				CheckedInputStream cis = new CheckedInputStream(uis, new CRC32())) {
 			// read bytes, minimum 1
-			long bytes = (nrbits * entries + 7) / 8;
+			long bytes = (getNrBits() * size() + 7) / 8;
```

Comment:
It looks like you have two methods called `size` where one returns a size and the other sets the size. Could the setter be called setSize()?

Reply:
Good point, I'll change it